new_dawn_server/locations/api/resources.py

```diff
@@ -1,5 +1,6 @@
 from django.conf.urls import url
-from new_dawn_server.locations.constants import us_city_mapping, country_list
+from new_dawn_server.locations.constants.city_constants import us_city_mapping, country_list
```

Comment:
Overall LGTM. 
A small style nit: all constant variables should be named with capital letters like US_CITY_MAPPING.